source/fe/mapping_fe_field.cc

```diff
@@ -496,15 +496,6 @@ MappingFEField<dim, spacedim, VectorType>::requires_update_flags(
            update_jacobian_pushed_forward_3rd_derivatives))
         out |= update_covariant_transformation;
 
-      // The contravariant transformation
-      // is a Piola transformation, which
-      // requires the determinant of the
-      // Jacobi matrix of the transformation.
-      // Therefore these values have to be
-      // updated for each cell.
-      if (out & update_contravariant_transformation)
-        out |= update_JxW_values;
-
```

Comment:
Separate PR for this change? The Piola transform uses `update_volume_elements` so I think the comment is outdated.